pkg/virt-handler/vm_test.go

```diff
@@ -949,12 +949,10 @@ var _ = Describe("VirtualMachineInstance", func() {
 				Message:   "",
 			}
 
-			vmiCopy := vmi.DeepCopy()
 			vmiFeeder.Add(vmi)
 			domainFeeder.Add(domain)
 
 			client.EXPECT().SyncVirtualMachine(vmi, gomock.Any())
-			vmiInterface.EXPECT().Update(NewVMICondMatcher(*vmiCopy))
```

Comment:
Note to the reviewer: its not fully clear to me why this expectation is set here while the description of the test states

> It("should do **nothing** if access credential condition already exists"

Reply:
it was likely a mistake